### DISCUSSION TITLE
Why did freeplane changed its hotkeys and how do I revert this?

### DISCUSSION THREAD

**Discussion by chumasoero** [freeplane/freeplane#972] [Help]
It's infuriating to enter a new version and see that many tyhings have changed (the hotkeys is a major problem for me) for the moment I've reverted to and old version.
I've tried to change manually some hotkeys to non avail.

Really, really infuriating.

**Comment by euu2021** [comment]
Please inform:

1. which version you were before updating
2. to which version you updated to
3. which hotkeys changed, or at least give an example
